packages/presentation/src/resources.ts

```diff
@@ -0,0 +1,8 @@
+import { Resources } from '@hcengineering/platform'
+import { StatusMiddleware } from './status'
+
+export default async (): Promise<Resources> => ({
```

Comment:
Presentation should not be loaded as plugin, I think we need to go try unify aggregator pipeline or move status aggregator to view plugin.